CMakeLists.txt

```diff
@@ -29,6 +29,10 @@ endif()
 if(LINUX)
   include(cmake/Rpm.cmake)
   include(cmake/Deb.cmake)
+  # RPATH seems to be a requirement when using Ninja as the cmake generator.
+  # as Ubuntu currently comes with 6.4.2, but the QtCreator is on 6.8 I'm pulling in the library from the Creator
+  set(CMAKE_BUILD_WITH_INSTALL_RPATH TRUE)
```

Comment:
I don't know why you need this, but this should'nt be needed, you can easily pass `-DCMAKE_BUILD_WITH_INSTALL_RPATH=TRUE` to cmake if you need this.

Reply:
removed